### PR TITLE
[ADD] amazon connector: features page

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: sales/amazon_connector/features.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Synchronization between Amazon and Odoo
+=======================================
+
+The **Amazon Connector** synchronizes the orders between Amazon and your Odoo database, which
+reduces considerably the amount of time spent on your Amazon Marketplace dashboard, making your
+daily routine a lot easier. More advanced tasks still require you to connect to your Amazon account.
+
+This connector is able to:
+
++--------------------------------------------------+----------------------------------------+
```
Comment (AntoineVDV) [inline sales/amazon_connector/features.rst:11]: I think that a table is not the format that we want here. We don't need a line-by-line comparison of Amazon -> Odoo vs Odoo -> Amazon but an enumeration of both flows' features.

Instead, a table could be used to compare FBA's features with FBM's features :
- orders -> FBA: sync shipped and canceled order ; FBM: sync unshipped and canceled orders (see manage.rst)
- shipping -> FBA: charges ; FBM: charges + delivery created
- gift -> FBA: nothing ; FBM: gift wrapping charges + gift message
- stock management -> FBA: 1 stock move created per SO line ; FBM: handled by the delivery
- confirmation -> FBA: nothing to do ; FBM: notify amazon when confirming delivery (or 'shipping', I can't ever remember the functional term)
- cancelation -> both: notify Amazon on manually canceled SO

File: sales/amazon_connector/features.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| From Amazon to Odoo                              | From Odoo to Amazon                    |
++==================================================+========================================+
+| Synchronize all confirmed orders (both FBA and   | Notify Amazon of a confirmed shipping  |
+| FBM) and their order items which include:        | (FBM) in order to get paid.            |
+|                                                  |                                        |
+| - the product's details (SKU, listing name &     |                                        |
```
Comment (AntoineVDV) [inline sales/amazon_connector/features.rst:17]: Let's drop the mention of product details. This is implicit and not even exhaustive anyway.

File: sales/amazon_connector/features.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+|   description, quantity, etc.)                   |                                        |
+| - the shipping costs for the product             |                                        |
+| - the gift wrapping charges                      |                                        |
++--------------------------------------------------+----------------------------------------+
+| Create any missing offer related to an order.    | Notify Amazon of a manually canceled   |
```
Comment (AntoineVDV) [inline sales/amazon_connector/features.rst:22]: That's 'technical' (sort of) and misleading since one could expect the offer to be created in Amazon, drop it 🚮

File: sales/amazon_connector/features.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+|                                                  | order (This feature has been dropped   |
+|                                                  | in Odoo 13.1 and higher versions)      |
```
Comment (AntoineVDV) [inline sales/amazon_connector/features.rst:24]: Let's not allow the doc to see into the future :p
We'll simply stop mentionning it when forward-porting the doc to 13.1+

File: sales/amazon_connector/features.rst (hunk 1 above)
Comment (AntoineVDV) [reply sales/amazon_connector/features.rst:11]: Since the list of features is no longer ->Odoo vs ->Amazon, let's insert before the comparisons between FBA and FBM that the connector supports multiple seller accounts and multiple marketplaces per seller account.

File: sales/amazon_connector/features.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
++--------------------------------------------------+----------------------------------------+
+| Create any missing partner related to an order   |                                        |
+| (contact types supported: contact and delivery). |                                        |
++--------------------------------------------------+----------------------------------------+
+
+.. note::
+   The connector handles the features listed above only and nothing else.
```
Comment (AntoineVDV) [inline sales/amazon_connector/features.rst:31]: Same as for "advanced tasks". Let's rather say something like "Downloading your montly fees report, handle disputes, issue refunds and other out of flow actions (rephrase this as you like^^) are not supported by the Amazon Connector. You should manage these in Amazon Seller Central."

File: sales/amazon_connector/features.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+.. seealso::
+   - :doc:`apply`
```
Comment (AntoineVDV) [inline sales/amazon_connector/features.rst:34]: Remove apply

File: sales/amazon_connector/manage.rst [hunk 7]
```diff
@@ -100,4 +100,9 @@ reporting for the sales of this account.
    remove the marketplace you wish to track separately from the list of synchronized marketplaces.
    Then, create another registration for this account and remove all marketplaces, except the one to
    isolate from the others. Finally, assign another sales team to one of the two registrations of
-   your account.
\ No newline at end of file
+   your account.
+
+.. seealso::
+   - :doc:`features`
+   - :doc:`apply`
```
Comment (AntoineVDV) [inline sales/amazon_connector/manage.rst:107]: Remove apply

File: sales/amazon_connector/setup.rst [hunk 8]
```diff
@@ -62,4 +62,9 @@ the subtotals between Seller Central and Odoo.
 .. note::
    As Amazon does not necessarily apply the same taxes as those configured in Odoo, it may happen
    that order totals differ by a few cents from that on Seller Central. Those differences can be
-   resolved with a write-off when reconciling the payments in Odoo.
\ No newline at end of file
+   resolved with a write-off when reconciling the payments in Odoo.
+
+.. seealso::
+   - :doc:`features`
+   - :doc:`apply`
```
Comment (AntoineVDV) [inline sales/amazon_connector/setup.rst:69]: Remove apply

File: sales/amazon_connector/features.rst (hunk 4 above)
Comment (jcs-odoo) [reply sales/amazon_connector/features.rst:24]: Should I add a todo:: then?

Comment (AntoineVDV) [reply sales/amazon_connector/features.rst:24]: Yes, why not

**Comment by AntoineVDV** [comment]
@jcs-odoo I think that you missed the hidden comments in the middle of the review ;)

File: sales/amazon_connector/features.rst (hunk 2 above)
Comment (jcs-odoo) [reply sales/amazon_connector/features.rst:17]: I removec the details between brackets but I think it's still informative :)
